README.md

```diff
@@ -88,7 +99,7 @@ Because `tap/3` uses `case` you will get a `CaseClauseError` with the data which
 
 #### Destruct
 
-Since `destruct/3` uses `=` you will instead get a `MatchError` with the data which did not match in the error report.
+Since `destruct/3` and `leak/2` use `=` you will instead get a `MatchError` with the data which did not match in the error report.
```

Comment:
~So, `leak/2` I don't believe can fail, can it?~

Looks like it can if you use a pattern, maybe we should add a note here saying that it won't fail for only variable names.